crates/bevy_text/src/lib.rs

```diff
@@ -52,6 +71,8 @@ impl Plugin for TextPlugin {
                 update_text2d_layout.after(ModifiesWindows),
             );
 
+        let _ = app.world.get_resource_or_insert_with(TextSettings::default);
```

Comment:
Why is this using `get_resource_or_insert_with`? Shouldn't this use `init_resource`

Reply:
Yes it should. looks like I needed to dig around the documentation some more. Fixed